tests/ui_tests/app/page/cluster/page.py

```diff
@@ -110,6 +115,19 @@ def check_all_elements(self):
             ]
         )
 
+    def click_add_service_btn(self):
+        self.find_and_click(ClusterServicesLocators.add_services_btn)
+
+    @allure.step("Check service {service_name} in cluster")
+    def add_service_by_name(self, service_name: str):
```

Comment:
The step is 'Check', but the method is 'add'
Looks more like "Add service {service_name} to cluster"

Reply:
yep, you're right, changed